sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentAccuracyRadiusSettingsInterface.kt

```diff
@@ -0,0 +1,35 @@
+package com.mapbox.maps.plugin.locationcomponent
+
+/**
+ * Interface that defines the public accuracy radius settings interface for LocationComponentPlugin.
+ */
+interface LocationComponentAccuracyRadiusSettingsInterface {
```

Comment:
similarly to above, shall we introduce a `LocationComponentSettingsInterface2` which is declared as 
```
interface LocationComponentSettingsInterface2 : LocationComponentSettingsInterface {
    fun getAccuracyRadiusSettings(): LocationComponentAccuracyRadiusSettings
    fun updateAccuracyRadiusSettings(block: LocationComponentAccuracyRadiusSettings.() -> Unit)
    var accuracyCircleEnabled: Boolean
    var accuracyCircleColor: Int
    var accuracyCircleAlpha: Float
}
```
to extend our existing interface?